tests/foreman/cli/test_user.py

```diff
@@ -106,10 +105,7 @@ def test_bugzilla_1079649_2(self):
 
     @data(
         {'login': gen_string("latin1", 10)},
-        {
-            'login': gen_string("utf8", 10),
-            'bugzilla': 1144162,
-        },
+        {'login': gen_string("utf8", 10)},
```

Comment:
I agree on dropping the default length, but I would suggest two approaches, adding a new commit to this PR or creating another one. I wanna maintain the history as clean as possible, and the purpose of this commit is to just unblock UTF8 related tests.